ci/scripts/makefile_to_install_yamls.py

```diff
@@ -0,0 +1,105 @@
+# Python script to automatically generate roles content for install_yamls
```

Comment:
While I fully support the concept of wrapper, I have some concerns:

since the goal is to call the various "make" from within ansible, via a single generated file, we'll get a hanging task for the whole time of the run. We'll therefore lose track of what's going on, and may lost important pieces in the process.

What about:
- generate an env-file with the needed ```export``` of the various environment variables we can set
- generate per-make call scripts, sourcing the generated env-file, maybe with some order prefix (1-foo, 2-bar, ...) in a dedicated directory
- call each of those script within ansible (we can loop on the directory content, no problem here)

That way, we'll get a more agile approach, more modular, and would allow to, maybe generate things but not run all of them (think "breakpoints support"). We'll also be able to properly manage the needed pauses, calling the related ```oc wait --for ... <resource>``` instead of static "sleep" in the script.

WDYT?

Reply:
Thank you @cjeanner for taking a look at the code. I liked the idea of moving stuffs in different files like:
* Moving the user exported vars in a single file
* Generating one file per make target
     * Then we can easily club it together based on user's need
It will make the role more modular and maintainable.

All these above stuffs can be achieved by the wrapper script. It will generate the role for us. We can also add custom logic where ever needed. Let me go ahead and modify my script and see how it goes.

Reply:
@raukadah Thanks for the answers , I'm +1 for @cjeanner's comment on running the individual generated makes from ansible and not bash.
Other than that I understand that some complexity in maintenance is necessary for this wrapper to be usable and reproducible across different envs. So I say let's go for it and see how it goes , nice job:)

I'm ok with this , please ask the core guys to approve this PR (Dan,Andrew,etc..)

Reply:
I'm cool with whatever you guys agree upon here, as long as it doesn't break the existing direct usage of the Makefile.